music/package.json

```diff
@@ -16,10 +16,13 @@
   },
   "devDependencies": {
     "@types/clone": "^0.1.30",
+    "@types/file-saver": "^1.3.0",
     "@types/tape": "^4.2.32",
     "browserify": "^14.4.0",
     "clang-format": "^1.2.3",
     "clone": "^1.0.4",
+    "file-saver": "^1.3.8",
+    "file-saver-typescript": "^1.0.1",
```

Comment:
do you need both? if you have file-saver and the @types?

Reply:
apparently, the answer is yes